ReadME.md

```diff
@@ -0,0 +1,12 @@
+## Jupyter ![](https://img.shields.io/github/forks/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/stars/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/watchers/TheAlgorithms/Jupyter?style=social) <br>
```

Comment:
Capitolization: `README.md` for consistency with other repos.

Reply:
Ok

Reply:
@cclauss kindly review the PR, it has been modified.